src/main/java/oilinjection/domain/car/Car.java

```diff
@@ -0,0 +1,41 @@
+package oilinjection.domain.car;
+
+import java.util.Objects;
+
+public abstract class Car {
+
+    final double tripDistance;
+
+    Car(final double tripDistance) {
+        this.tripDistance = tripDistance;
+    }
+
+    abstract double getDistancePerLiter();
+
+    public abstract String getType();
+
+    double getTripDistance() {
+        return this.tripDistance;
+    }
+
+    public double getChargeQuantity() {
+        return Math.ceil(getTripDistance() / getDistancePerLiter());
```

Comment:
객체안에서는 메서드가 아닌 상태를 바로 호출해주면 어떨까요?

```suggestion
        return Math.ceil(this.tripDistance / getDistancePerLiter());
```

Reply:
아하! 넵 감사합니다! 멤버변수로 갖고 있는 것을 굳이 getter로 호출할 필요가 없네요!